### DISCUSSION TITLE
Add  void Post(Action<object> action,object arg, DispatcherPriority priority = default);

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#8298] [Ideas]
In commit 4ec36c97e2810c8e1bb48f85339d76e96e502b78 of PR #8021 are remove generic methods from IDispatcher. 
Post<T> was added in PR #6527 to avoid closures in intense operations such as:

https://github.com/AvaloniaUI/Avalonia/blob/3e3396d132385fd1304fc2ddef873911acd273ca/src/Avalonia.Visuals/Rendering/RenderLoop.cs#L112-L136

I would like to propose you to add the non-generic version which accepts an argument of type object.

**Comment by maxkatz6** [comment]
It was done https://github.com/AvaloniaUI/Avalonia/pull/8304
